action.yaml

```diff
@@ -12,6 +12,10 @@ inputs:
     description: 'Path to composer cache'
     required: true
     default: /tmp/composer-cache
+  ignore-platform-reqs:
```

Comment:
Either we should make this more generic (`composer-options:` with this default), or we should only use the value of this option, and not include the option name here (`--ignore-platform-req=${{ inputs.ignore-platform-reqs }}`). That will make it more suitable for different purposes, without compromising what you're intending to do here.

Reply:
Will do the first. FYI, if you want to ignore multiple reqs then you'll have to include multipe `--ignore-platform-req` you cannot comma separate them.

Reply:
Ah right, in that case I think having an input to specify the entire list of options seems preferable indeed.